compiler/src/iree/compiler/Dialect/Util/Transforms/ConvertPrimitiveType.cpp

```diff
@@ -210,11 +210,108 @@ struct ConvertTypeSensitiveArithCastOp : public OpConversionPattern<OpTy> {
   }
 };
 
+// Truncates input operand if it is a constant and is out of range of target
+// type.
+Value truncateOutOfRangeValue(Value operand,
+                              ConversionPatternRewriter &rewriter,
+                              TypeConverter *typeConverter) {
+  if (!operand.getDefiningOp()) {
```

Comment:
`auto constOp = operand.getDefininingOp<arith::ConstantOp>()` here.

Reply:
Thanks! was about to use dyn_cast_or_null, but this is much better :)